packages/common/src/search/index.ts

```diff
@@ -10,4 +10,5 @@ export * from "./types";
 export * from "./collectionState";
 export * from "./convertCatalog";
 export * from "./filter-utils";
+export * from "./ifilter-utils";
```

Comment:
do these need to be exported to external users of the library?

Reply:
I'll check on this - I think there are a few fn's that may make sense, but I'll see if I can avoid it